argo/apps/templates/sbat-iam-init-config.yaml

```diff
@@ -0,0 +1,27 @@
+# This initializer pod will be deployed in the namespace with the environment type = 'FIDC'
+# Otherwise, currently this pod will be deployed in cdk namespace only
+# @See the pipeline initialise, step 'install_initializer_cron_job'
+
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: iam-initialiser-config-{{ .Release.Name }}
```

Comment:
This will need to change is we give the chart a good name (see https://github.com/SecureApiGateway/sapig-openbanking-uk-developer-envs/pull/3)